models/rate_neuron_ipn.h

```diff
@@ -178,6 +178,9 @@ class rate_neuron_ipn : public Archiving_Node
     /** Mean input.*/
     double mu_;
 
+    /** Minimum rate.*/
+    double rectify_rate_;
```

Comment:
I think that l. 191 needs to be changed:
`True: If the rate is negative it is set to zero after each time step.` &rarr; `True: If the rate is smaller than rectify_rate it is set to rectify_rate after each time step.`.

Reply:
Thanks @jstapmanns . You are right! Your suggestion has been applied!